opal/mca/hwloc/external/configure.m4

```diff
@@ -172,21 +197,9 @@ AC_DEFUN([MCA_opal_hwloc_external_CONFIG],[
 
            AC_CHECK_HEADERS([infiniband/verbs.h])
 
-           AC_MSG_CHECKING([if external hwloc version is 1.5 or greater])
            AS_IF([test "$opal_hwloc_dir" != ""],
                  [opal_hwloc_external_CFLAGS_save=$CFLAGS
                   CFLAGS="-I$opal_hwloc_dir/include $opal_hwloc_external_CFLAGS_save"])
-           AC_COMPILE_IFELSE(
-               [AC_LANG_PROGRAM([[#include <hwloc.h>]],
-                   [[
-#if HWLOC_API_VERSION < 0x00010500
-#error "hwloc API version is less than 0x00010500"
-#endif
-                   ]])],
-               [AC_MSG_RESULT([yes])],
-               [AC_MSG_RESULT([no])
-                AC_MSG_ERROR([Cannot continue])])
-
```

Comment:
Per #5031, if external support is not available, we need to emit a message at the bottom of this configure.m4:

1. If the user explicitly asked for external support, print why the external hwloc is not suitable and then abort.
2. If the user did not explicitly ask for external support, print why the external hwloc is not suitable and then allow processing to continue (which will end up selecting the internal hwloc).

Reply:
This is fixed in the latest commit